src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.ts

```diff
@@ -88,13 +88,14 @@ export class EditKeyboardMappingComponent implements OnInit {
     return this.selectedAction.argument.type
   }
 
-  public save(): void {
+  public save(isSaveAndCreateNew?: boolean): void {
     const keyboardMapping: KeyboardMapping = {
       action: {} as Tv2PartAction,
       actionTrigger: {
         ...this.keyboardMapping?.actionTrigger,
         ...this.actionTriggerForm.value,
       },
+      isSaveAndCreateNew,
```

Comment:
This newly introduced attribute has nothing to do with the KeyboardMapping type. If we want to overload the `onSave` output, then we should use composition, such that the type becomes something like
`{ shouldSaveAndCreateNew: boolean, keyboardMapping: KeyboardMapping }`.

An alternative is to have two separate outputs for the two type of actions. E.g. `onSave` and `onSaveAndCreateNew`.

Reply:
@KvelaGorrrrnio event separate the outputs I don't see the way to separate the callback to afterClosed `method `:

 ```
 public createConfirmDialog(
    title: string,
    message: string,
    okButtonText: string,
    onOk: () => void,
    theme: DialogColorScheme = DialogColorScheme.DARK,
    severity: DialogSeverity = DialogSeverity.INFO
  ): void {
    this.open(ConfirmationDialogComponent, {
      data: {
        title: title,
        message: message,
        buttonText: {
          ok: okButtonText,
          cancel: $localize`dialog.cancel-button:Cancel`,
        },
        theme,
        severity,
      },
    })
      .afterClosed()
      .subscribe(result => {
        if (result) {
          onOk()
        }
      })
  }
```
I will change it to:
`{ shouldSaveAndCreateNew: boolean, keyboardMapping: KeyboardMapping }`

Reply:
Done